04-ruby-mvc-sinatra/databases/sql_basics.md

```diff
@@ -0,0 +1,333 @@
+# Databases
+
+## Objectives
+
+### Concepts
+
+- Explain what a database is and why you would use one as opposed to other persistent storage mechanisms
+- Explain the difference between a database management system (DBMS) and a database, and name the major DBMSes
+- Explain how a DBMS, a database, and SQL relate to one another
+- Describe a database schema and how it relates to tables, rows and columns
+- List common data types used in SQL and the related field constraints
+- Describe what a SQL injection attack is
+
+### Mechanics
+
+- Create a new PostgreSQL database
+- Set up a PostgreSQL database schema with a saved SQL file
+- Seed a PostgreSQL database with a saved SQL file
+- Execute basic SQL commands to execute CRUD actions in a database
+
```

Comment:
add opening framing section. what are we talking about today? why now? how does this relate to what came before / what's to come?